index.md

```diff
@@ -52,8 +52,29 @@ because your past self doesn't answer email.
+
+**Be the raw data you wish to see in the world.** From the raw data, make the dataset you *wish* you had received and that you will enjoy having as the start point for downstream analyses, some of which you have not even dreamed of yet. This is the place to maximize machine and human readability, preferably at the same time. It is neither the place to do vigorous data filtering nor to the time to bring in external information. Approach this initial preparation as internal to the existing dataset and non-destructive, both of the data and its general "shape". Enhance machine readability by converting from proprietary and high-friction formats (e.g., Microsoft Excel or XML) to open and simple formats (e.g., comma delimited plain text). Enhance human readability by replacing inscrutable variable names and artificial data codes with self-explaining alternatives. For example, rename the variable SOME_AWFUL_GENERIC_NAME to SOMETHING_DESCRIPTIVE *(JB: anyone ... got a good example that's real/realistic but will resonate with many?)*, recode the treatment variable from `1` vs. `2` to `untreated` vs. `treated`, and replace artificial codes, such as "-99" for missing data, with proper `NA`s. Both human and machine readability can be enhanced storing especially useful metadata as part of the filename itself, while keeping the filename regular-expression-friendly.
+
+**Create analysis-friendly data.** Well-prepared raw data can still present many barriers to analysis and visualization. For example, columns in the dataset may not represent the variables that are most conducive to analysis. There are (at least!) two classes of problems:
+
+  1. Columns that contain more than one variable's worth of information. For example, the inclusion of units is problematic, e.g., "3.4 kg". The presence of "kg" will cause most analytical environments to read this in as character data, whereas you'll probably want to do numeric things with it, like take averages and use in plots. This should be split into two variables, the mass "3.4" and the units "kg" (or the units should be recorded in the variable name and/or metadata). When in doubt, try to make each variable correspond to an atomic, imminently usable piece of information.
+  2. Multiple columns that -- only when taken together -- contain one variable's worth of information. This is characteristic of data that has been laid out for human eyeballs or for manual data entry. For example, there might be one row per field site and then columns for measurements made at each of several time points. For data entry and inspection, it is convenient to store this in a "short and wide" form, but for most analyses it will be advantageous to gather these columns into a variable of measurements, accompanied by a companion variable indicating the time point.
```

Comment:
Add two example tables, one showing human-oriented "short and wide", the other what we want.